examples/saul/Makefile

```diff
@@ -0,0 +1,28 @@
+# name of your application
+APPLICATION = default
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# we want to use SAUL:
+USEMODULE += saul_reg
+USEMODULE += saul_default
```

Comment:
yepp, that could be an option, didn't actually check if this is already the case...